ci-operator/config/openshift/logging-view-plugin/openshift-logging-view-plugin-main.yaml

```diff
@@ -0,0 +1,29 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- dockerfile_literal: |
+    FROM cypress/included:3.8.0
```

Comment:
This base image should be a RH base given we are likely to push the final to quay and need to justify security